### PR TITLE
Fix coding standards

### PR DESCRIPTION
**Pull request by joemcgill**
* Adds phpcs.xml rule for the project.
* Applies Alley Coding Standards rule updates via phpcbf and manual fixes.
* Updates .travis.yml to make style rules required.

### CODE REVIEW

File: asset-manager.php [hunk 1]
```diff
@@ -59,34 +60,36 @@ function am_enqueue_script( $handle, $src = false, $deps = array(), $condition =
...
-	 * @param string $handle 	  Handle for stylesheet. This is necessary for dependency management.
-	 * @param string $src 		  URI to stylesheet.
+	 * @param string $handle      Handle for stylesheet. This is necessary for dependency management.
+	 * @param string $src         URI to stylesheet.
+	 * @param array  $deps        List of dependencies.
 	 * @param string $condition   Corresponds to a configured loading condition that, if matches, will allow the stylesheet to load. Defaults are 'global', 'single', and 'search'.
 	 * @param string $load_method How to load this asset.
-	 * @param string $load_hook   Hook on which to load this asset
-	 * @param string $media 	  Media query to restrict when this asset is loaded
+	 * @param string $version      Version of the script.
```
Comment (goodguyry) [inline asset-manager.php]: Should the CS have caught this, or is the alignment not required?

```suggestion
	 * @param string $version     Version of the script.
```

File: php/class-asset-manager-scripts.php [hunk 2]
```diff
@@ -93,38 +99,38 @@ public function set_asset_type_defaults() {
...
 	}
 
 	/**
-	 * Disable JS concatenation for async and defer scripts
+	 * Disable JS concatenation for async and defer scripts.
 	 *
-	 * @param bool $do_concat Whether or not to concatenate this script
-	 * @param string $handle  Handle for enqueued script
+	 * @param bool   $do_concat Whether or not to concatenate this script.
+	 * @param string $handle  Handle for enqueued script.
```
Comment (goodguyry) [inline php/class-asset-manager-scripts.php]: Related to my other question :)

```suggestion
	 * @param bool   $do_concat Whether or not to concatenate this script.
	 * @param string $handle    Handle for enqueued script.
```

**Review by goodguyry — APPROVED** [review]
Just one question (asked twice) 🎸

File: asset-manager.php (hunk 1 above)
Comment (joemcgill) [reply asset-manager.php]: Interesting. It looks like the WordPress standards explicitly don't include this sniff. TIL

```
<!-- Excluded to allow param documentation for arrays  -->
<exclude name="Squiz.Commenting.FunctionComment.SpacingAfterParamName"/>
```

**Pull request merged**